### ISSUE TITLE
Steamshovel does not build as C++11 with clang (Trac #1593)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1585]
Trying to compile steamshovel with "Apple LLVM version 7.0.2 (clang-700.1.81)" or "clang version 3.9.0 (trunk 263374)" in C++11 mode yields the an error:

```text
combo/src/steamshovel/private/scripting/qmeta_args.cpp:123:11:
error: no matching constructor for initialization of 'QVariant'
```

It is unclear to me why this error occurs and whether this is a clang bug or a Qt bug, however, it can be worked around by:

```text
Index: private/scripting/qmeta_args.cpp
===================================================================
--- private/scripting/qmeta_args.cpp	(revision 143308)
+++ private/scripting/qmeta_args.cpp	(working copy)
@@ -120,7 +120,9 @@
 void property_write( QMetaProperty p, QObject* q, T value )
 {
 	log_trace_stream("Writing property " << p.name() << " of object " << q );
-	QVariant v( value );
+	QVariant v;
+	v.setValue(value);
 	bool success = false;
 	if( q->thread() == QThread::currentThread() ){
```

I can post a more complete error message if necessary/useful. 

Migrated from https://code.icecube.wisc.edu/ticket/1593
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T14:21:45", 
    "description": "Trying to compile steamshovel with \"Apple LLVM version 7.0.2 (clang-700.1.81)\" or \"clang version 3.9.0 (trunk 263374)\" in C++11 mode yields the an error:\n\n{{{\ncombo/src/steamshovel/private/scripting/qmeta_args.cpp:123:11:\nerror: no matching constructor for initialization of 'QVariant'\n}}}\n\nIt is unclear to me why this error occurs and whether this is a clang bug or a Qt bug, however, it can be worked around by:\n\n{{{\nIndex: private/scripting/qmeta_args.cpp\n===================================================================\n--- private/scripting/qmeta_args.cpp\t(revision 143308)\n+++ private/scripting/qmeta_args.cpp\t(working copy)\n@@ -120,7 +120,9 @@\n void property_write( QMetaProperty p, QObject* q, T value )\n {\n \tlog_trace_stream(\"Writing property \" << p.name() << \" of object \" << q );\n-\tQVariant v( value );\n+\tQVariant v;\n+\tv.setValue(value);\n \tbool success = false;\n \tif( q->thread() == QThread::currentThread() ){\n}}}\n\nI can post a more complete error message if necessary/useful. ", 
    "reporter": "cweaver", 
    "cc": "hdembinski", 
    "resolution": "fixed", 
    "_ts": "1458310905077210", 
    "component": "combo reconstruction", 
    "summary": "Steamshovel does not build as C++11 with clang", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-03-18T06:15:01", 
    "milestone": "", 
    "owner": "cweaver", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160318T13:52:35: hdembinski commented:

> Hi Chris,
> 
> please go ahead and commit that. :)

**Comment by icecube-trac** [comment]
Trac update at 20160318T14:21:45: 
* cweaver changed owner from "" to "cweaver"
* cweaver changed status from "new" to "closed"
* cweaver commented:

> In changeset 143404:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="143404"
> Work around a C++11 problem. Fixes #1585.
> ```


* cweaver changed resolution from "" to "fixed"
